### ISSUE TITLE
Stop calling our input format "texlike" and just call it "sil"

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#1055]
I'm starting to get annoyed at the awkwardness of ‘texlike’ (and its various iterations of capitalization) in the code and in talking about SILE. I understand why this moniker is useful (for some select people) in helping to understand what the language is *like*; but it is not TeX nor ever that similar to TeX. I've come to the conclusion that it would be better to stop using this term entirely, especially in the code but also just in official references to the format. We can certainly mention in documentation and in prose that the SIL input format has some similarity to TeX, but the format itself shouldn't be named after TeX. I'm convinced that perpetually using the name of a format we're technically not related to makes it harder both for non-TeX folks (who have no idea what we are talking about) and TeX folks (who tend to assume it's more similar than it is) to actually grok in the long run.

Lets just call it "SIL" input and "XML" input and end the charade.

**Comment by alerque** [comment]
Tangentially related to #508.

**Comment by Omikhleia** [comment]
> Lets just call it "SIL" input

"SILE scripting language" perhaps (or "SILE script" in short). The acronym "SIL" refers to something else (an organization) and might be confusing.

**Comment by alerque** [comment]
It isn't a scripting language (that would be Lua), it is a declarative markup language. I'm also afraid "SILE markup" would confuse people because XML is a different input format but actually has the same "markup" language as SIL input.

In any case this is kind of a moot point because I've already almost completely excised "texlike" from the code and documentation. Functions like `SILE.doTexlike("foo")` are deprecated in favor of `SILE.processString("foo", "sil")`, and the documentation has been updated (all but one reference is chapter 12 I seem to have missed).

I'm happy to consider what the best way to talk about `*.sil` files is, but we've already settled on not commonly referring to them as "texlike".
